lib/Sender.js

```diff
@@ -78,7 +78,7 @@ class Sender {
    */
   doPing (data, options) {
     var mask = options && options.mask;
-    this.frameAndSend(0x9, data || '', true, mask);
+    this.frameAndSend(0x9, data ? Buffer.from(data.toString()) : null, true, mask);
```

Comment:
Ok, It might be better to use `String(data)` which won't fail with `null` or `undefined`.